README.md

```diff
@@ -1,313 +1,386 @@
 
-	setupViewTransition();
+  setupViewTransition();
 </script>
 
 <slot />
+
+<style>
+  /* Disable default crossfade. */
+  :root {
+    view-transition-name: none;
+  }
+
+  /* Or, just modify the duration. */
+  ::view-transition-old(root),
+  ::view-transition-new(root) {
+    animation-duration: 2s;
+  }
+</style>
 ```
 
-You can target and modify the default animation with `::view-transition-old(root)` and `::view-transition-new(root)`. However it's ofter useful to have specific part of the page have different view transitions. For example the header might be fixed and we would like to animate it differently. To allow for this use case `setupViewTransition` return an object from which you can destructure `transition`.
+<br>
+
+## `transition`
+
+It's often useful to give specific parts of the page their own unique view transitions.  We can do this by setting an element's `view-transition-name`.
 
-### transition
+One way to do this is with `transition`, a svelte [action](https://svelte.dev/tutorial/actions) returned by `setupViewTransition`.
 
-`transition` is an action that accept a string as input and assign that string as the `view-transition-name` of that element. Plain and simple.
+`transition` accepts a string representing the `view-transition-name` that should be assigned to the element using it:
 
 ```svelte
 <script>
-	import { setupViewTransition } from 'sveltekit-view-transition';
+  import { setupViewTransition } from 'sveltekit-view-transition';
 
-	const { transition } = setupViewTransition();
+  const { transition } = setupViewTransition();
 </script>
 
 <header use:transition={'header'}>
-	<!-- links -->
+  <!-- links -->
 </header>
+
 <slot />
+
+<style>
+  ::view-transition-old(header),
+  ::view-transition-new(header) {
+    /* ... */
+  }
+</style>
 ```
 
-This will allow you to target the header with `::view-transition-old(header)` and `::view-transition-new(header)`.
+As you can see, the header's view transition can now be modified via `::view-transition-old(header)` and `::view-transition-new(header)`.
 
-Now while this may seem enough sometimes you might want to be a bit more creative with the transition names and that's why instead of a string you can also pass an object to the transition action.
+<br>
 
-#### name
+## Additional Options
 
-This is the name of the transition, much like the string that you would've passed before it's what will end up as the `view-transition-name`. It's the only required field and it can be either a string or a function that takes a navigation object and return a string.
+If you want to be a bit more creative with the transitions, you can pass an `object` to the `transition` action instead of a `string`.  This object accepts the following options:
 
-#### classes
+- `name`: the view-transition-name *(required)*
+- `classes`: classnames to apply to the target element during the transition.  An array of strings or a function that returns one.
```

Comment:
It's not the target element, it's the `root` since view transitions are always child of the `root`